internal/kafka/internal/handlers/admin_kafka_test.go

```diff
@@ -477,10 +484,155 @@ func Test_adminKafkaHandler_Update(t *testing.T) {
+						return true, nil
+					},
+					CheckStrimziVersionReadyFunc: func(cluster *api.Cluster, strimziVersion string) (bool, error) {
+						return true, nil
+					},
+				},
+				kafkaService: &services.KafkaServiceMock{
+					GetFunc: func(ctx context.Context, id string) (*dbapi.KafkaRequest, *errors.ServiceError) {
+						return &dbapi.KafkaRequest{
+							Status: constants.KafkaRequestStatusSuspending.String(),
+							Meta: api.Meta{
+								ID: "id",
+							},
+							ClusterID:              "cluster-id",
+							ActualKafkaIBPVersion:  "2.8",
+							DesiredKafkaIBPVersion: "2.8",
+							ActualKafkaVersion:     "2.8",
+							DesiredKafkaVersion:    "2.8",
+							DesiredStrimziVersion:  "2.8",
+							KafkaStorageSize:       "100",
+						}, nil
+					},
+					VerifyAndUpdateKafkaAdminFunc: func(ctx context.Context, kafkaRequest *dbapi.KafkaRequest) *errors.ServiceError {
+						return nil
+					},
+				},
+				accountService: account.NewMockAccountService(),
+			},
+			args: args{
+				url:  kafkaByIdUrl,
+				body: []byte(`{"suspended": false}`),
+			},
+			wantStatusCode:  http.StatusOK,
+			wantKafkaStatus: constants.KafkaRequestStatusResuming,
+		},
+		{
+			name: "should have no effect on not suspending/suspended kafka Status if suspended param is set to false",
+			fields: fields{
+				clusterService: &services.ClusterServiceMock{
+					FindClusterByIDFunc: func(clusterID string) (*api.Cluster, *errors.ServiceError) {
+						return &api.Cluster{
+							Meta: api.Meta{
+								ID: "id",
+							},
+							ClusterID: clusterID,
+						}, nil
+					},
+					IsStrimziKafkaVersionAvailableInClusterFunc: func(cluster *api.Cluster, strimziVersion, kafkaVersion, ibpVersion string) (bool, error) {
+						return true, nil
+					},
+					CheckStrimziVersionReadyFunc: func(cluster *api.Cluster, strimziVersion string) (bool, error) {
+						return true, nil
+					},
+				},
+				kafkaService: &services.KafkaServiceMock{
+					GetFunc: func(ctx context.Context, id string) (*dbapi.KafkaRequest, *errors.ServiceError) {
+						return &dbapi.KafkaRequest{
+							Status: constants.KafkaRequestStatusReady.String(),
+							Meta: api.Meta{
+								ID: "id",
+							},
+							ClusterID:              "cluster-id",
+							ActualKafkaIBPVersion:  "2.8",
+							DesiredKafkaIBPVersion: "2.8",
+							ActualKafkaVersion:     "2.8",
+							DesiredKafkaVersion:    "2.8",
+							DesiredStrimziVersion:  "2.8",
+							KafkaStorageSize:       "100",
+						}, nil
+					},
+					VerifyAndUpdateKafkaAdminFunc: func(ctx context.Context, kafkaRequest *dbapi.KafkaRequest) *errors.ServiceError {
+						return nil
+					},
+				},
+				accountService: account.NewMockAccountService(),
+			},
+			args: args{
+				url:  kafkaByIdUrl,
+				body: []byte(`{"suspended": false}`),
+			},
+			wantStatusCode:  http.StatusOK,
+			wantKafkaStatus: constants.KafkaRequestStatusReady,
```

Comment:
Can we also add a test case here to ensure that status is not affected if suspended was not specified in the request?